build/config.cmake

```diff
@@ -3,8 +3,11 @@ CMAKE_MINIMUM_REQUIRED (VERSION 2.8)
 #######################################################
 # CONFIGURATION
 #######################################################
-SET (ESCARGOT_CXXFLAGS_CONFIG)
-SET (ESCARGOT_LDFLAGS_CONFIG)
+IF (EXISTS "build/cmake/toolchain_${ESCARGOT_HOST}_${ESCARGOT_ARCH}.cmake")
+    INCLUDE ("build/cmake/toolchain_${ESCARGOT_HOST}_${ESCARGOT_ARCH}.cmake")
```

Comment:
Please change the file path based on the escargot source directory such as `${ESCARGOT_ROOT}/build/cmake/toolchain_${ESCARGOT_HOST}_${ESCARGOT_ARCH}.cmake`
Others are fine.

Reply:
Thanks! I applied your suggestion.